spectral_cube/tests/test_stokes_spectral_cube.py

```diff
@@ -45,21 +51,47 @@ def test_direct_init_invalid_shape(self, use_dask):
             cube = StokesSpectralCube(stokes_data)
         assert exc.value.args[0] == "All spectral cubes should have the same shape"
 
-    @pytest.mark.parametrize('component', VALID_STOKES)
-
+    @pytest.mark.parametrize('component', VALID_STOKES_LIST)
     def test_valid_component_name(self, component, use_dask):
-        stokes_data = {component: SpectralCube(self.data[0], wcs=self.wcs, use_dask=use_dask)}
-        cube = StokesSpectralCube(stokes_data)
-        assert cube.components == [component]
+        # Register custom symbols if needed
+        custom_map = {
+            -9: StokesSymbol('RX', 'Custom RX'),
+            -10: StokesSymbol('RY', 'Custom RY'),
+            -11: StokesSymbol('LX', 'Custom LX'),
+            -12: StokesSymbol('LY', 'Custom LY'),
+            -13: StokesSymbol('XR', 'Custom XR'),
+            -14: StokesSymbol('XL', 'Custom XL'),
+            -15: StokesSymbol('YR', 'Custom YR'),
+            -16: StokesSymbol('YL', 'Custom YL'),
+            -17: StokesSymbol('PP', 'Custom PP'),
+            -18: StokesSymbol('PQ', 'Custom PQ'),
+            -19: StokesSymbol('QP', 'Custom QP'),
+            -20: StokesSymbol('QQ', 'Custom QQ'),
+            -21: StokesSymbol('RCircular', 'Custom RCircular'),
+            -22: StokesSymbol('LCircular', 'Custom LCircular'),
+            -23: StokesSymbol('Linear', 'Custom Linear'),
+            -24: StokesSymbol('Ptotal', 'Custom Ptotal'),
+            -25: StokesSymbol('Plinear', 'Custom Plinear'),
+            -26: StokesSymbol('PFtotal', 'Custom PFtotal'),
+            -27: StokesSymbol('PFlinear', 'Custom PFlinear'),
+            -28: StokesSymbol('Pangle', 'Custom Pangle'),
+        }
```

Comment:
is this a repeat of _custom_stokes_map?  We should import it in that case

Reply:
I made that change and pushed in already